modules/apps/29-fee/keeper/grpc_query_test.go

```diff
@@ -120,11 +120,11 @@ func (suite *KeeperTestSuite) TestQueryIncentivizedPackets() {
 				fee3 := types.NewIdentifiedPacketFee(channeltypes.NewPacketId(ibctesting.FirstChannelID, transfertypes.PortID, 3), fee, refundAcc.String(), []string(nil))
 
 				expPackets = []*types.IdentifiedPacketFee{}
```

Comment:
```suggestion
				expPackets = []types.IdentifiedPacketFee{}
```

You can make this a slice of values instead of pointers too and avoid the dereference on L:127

Reply:
@colin-axner spoke with damian offline and did not change this as I'm using expPackets later to compare with the grpc response. It makes sense to keep it as is.